lib/ReactTags.js

```diff
@@ -95,9 +95,9 @@ class ReactTags extends Component {
   handleDelete(i, e) {
     this.props.handleDelete(i, e);
     if (!this.props.resetInputOnDelete) {
-        this.textInput.focus();
```

Comment:
Would recommend running this through prettier `npm run format` to make sure we are consistent formatting / linting wise.

Reply:
I ran `npm run format` which resulted in this change. Might be  `npm run format ` was not run when the above code was pushed.